lib/teleterm/clusters/storage.go

```diff
@@ -42,9 +42,7 @@ func NewStorage(cfg Config) (*Storage, error) {
 
 // ListProfileNames returns just the names of profiles in s.Dir.
 func (s *Storage) ListProfileNames() ([]string, error) {
-	profileStore := client.NewFSProfileStore(s.Dir)
-	pfNames, err := profileStore.ListProfiles()
-	return pfNames, trace.Wrap(err)
+	return s.ClientStore.ListProfiles()
```

Comment:
Should we wrap the error here?

Reply:
If the function already wraps the error, then there's no reason to wrap it unless we want to add a new message. See below we `trace.Wrap` just returns the existing error.

```
	if traceErr, ok := err.(Error); ok {
		trace = traceErr
	} else {
		trace = newTrace(err)
	}
```

Reply:
All this time I've thought that `trace.Wrap` is important so that we can capture the stack trace. 🤦 But this is of course done with the runtime package.